sections/title.py

```diff
@@ -75,7 +55,7 @@ def contents():
             ),
             html.Li(
                 [
-                    'Rethink Priorities survey results: ',
+                    'Rethink Priorities Survey: ',
```

Comment:
Come to think of it, we should just call it "EA Survey:"

Reply:
Should I re-write it as 'EA Survey' or keep it the same?